code/game/gamemodes/meteor/meteors.dm

```diff
@@ -256,7 +256,9 @@
 	meteordrop = /obj/item/weapon/ore/uranium
 
 /obj/effect/meteor/irradiated/meteor_effect()
-	new /obj/effect/decal/cleanable/greenglow(get_turf(src)) //todo: make this irradiate the place it lands
+	new /obj/effect/decal/cleanable/greenglow(get_turf(src))
+	for(var/mob/living/L in view(5, src))

```

Comment:
Yes it will, but ,take in mind that meteor effect also happens with an explosion.